server/routes/api.js

```diff
@@ -31,6 +35,39 @@ router.route('/users')
       });
   });
 
+router.post('/register', (req, res) => {
+  bcrypt.genSalt(saltRounds, (err, salt) => {
+    if (err) { console.log(err); }
```

Comment:
Instead of doing console.logs, you should handle this error in a way that the request to register won't hang.